src/MagnumExternal/OpenGL/GL/flextGL.cpp

```diff
@@ -30,7 +30,7 @@ FLEXTGL_EXPORT void(APIENTRY *flextglMultiDrawArraysIndirectCountARB)(GLenum, GL
 FLEXTGL_EXPORT void(APIENTRY *flextglMultiDrawElementsIndirectCountARB)(GLenum, GLenum, GLintptr, GLintptr, GLsizei, GLsizei) = nullptr;
 
 /* GL_ARB_robustness */
-FLEXTGL_EXPORT GLenum(APIENTRY *flextglGetGraphicsResetStatusARB)(void) = nullptr;
+FLEXTGL_EXPORT GLenum(APIENTRY *flextglGetGraphicsResetStatusARB)() = nullptr;
```

Comment:
These files are autogenerated by flextGL, so this is just what the GL XML spec files say. I could change flextGL to don't put `void` there, but I don't think it's worth the effort :)